docs/specifications/tests/DNSSEC-TP/dnssec10.md

```diff
@@ -24,115 +39,204 @@ The description of the NSEC3 RR is in
 use in the DNS response is described in
 [RFC 5155][RFC 5155#section-7.2], section 7.2.
 
+
 ## Scope
 
-It is assumed that *Child Zone* has been tested by [Basic04]. This test
-case will set DEBUG level on messages for non-responsive name servers.
+It is assumed that *Child Zone* is tested by [Basic04]. Isues covered by
+[Basic04] (basic name server issues) will no result in messagess from this test
+case.
```

Comment:
> It is hard to be completely consistent.

I totally agree this is not an easy task. TBH as I was reading the specs one after another I thought it would be nice to try to keep similar parts consistent.